packages/ragbits-cli/src/ragbits/cli/state.py

```diff
@@ -0,0 +1,64 @@
+import json
```

Comment:
Note: the code in this file is almost entirely copied from now deleted `cli/app.py`, with the main difference of the state/functions being standalone and using global state instead of being a part of a class inheriting from `typer.Typer`.

The only other difference is the typing of `data` argument for `print_output` changing from `list[BaseModel] | BaseModel` (mostly to avoid problems with `list` being invariant).